app/main.py

```diff
@@ -289,6 +314,23 @@ def update_live_alerts_data(
     # And we deduce the subset of alerts that we can deem to be "live"
     live_alerts = all_alerts[mask_acknowledgement].copy()
 
+    # We then fetch sunrise and sunset times and add a safety margin of 30 min (converting from UTC) to cover night time
+    sunrise = night_time_data['results']['sunrise'][:-6]
+    sunrise = datetime.fromisoformat(str(sunrise)) + timedelta(hours=2.5)
```

Comment:
If the hours returned by `api.sunrise-sunset` are in UTC too, wouldn't it be simpler not to make any conversion to GMT (neither in the `is_hour_between` function nor here)?

Reply:
You're right but as we need to use timedelta to add the 30 min margin this is not an issue to leave it like this i guess